app/views/menu.html

```diff
@@ -1,5 +1,7 @@
 <ul class="nav navbar-nav">
   <li class="active"><a href="#/">Home</a></li>
+  <li><a ng=href="#/classify">Rating game</a></li>
```

Comment:
This is how the menu was made by Gijs.
I sticked to the structure as you can see.
I just added another button, the uit-sref is in index.html.

There should be a reason why it's this way, I think. 
You should ask @GijsWeterings